shell/src/main/java/alluxio/shell/command/CopyToLocalCommand.java

```diff
@@ -171,8 +172,9 @@ private void copyToLocal(AlluxioURI srcPath, File dstFile) throws IOException {
    */
   private void copyFileToLocal(AlluxioURI srcPath, File dstFile) throws IOException {
     try {
-      File tmpDst = File.createTempFile("copyToLocal", null);
-      tmpDst.deleteOnExit();
+      String randomSuffix = String.format(".%s_copyToLocal_",
+          RandomStringUtils.randomAlphanumeric(8));
```

Comment:
Nope, randomAlphanumeric characters will be chosen from the set of alpha-numeric characters.